packages/core/content-releases/server/src/middlewares/documents.ts

```diff
@@ -27,7 +28,7 @@ const updateActionsStatusAndUpdateReleaseStatus = async (
 
   const entryStatus = await isEntryValid(contentType, entry, { strapi });
 
-  await strapi.db.query(RELEASE_ACTION_MODEL_UID).update({
+  await strapi.db.query(RELEASE_ACTION_MODEL_UID).updateMany({
```

Comment:
you can have two actions refering to the published and draft version of a document?

Reply:
No, but you could have the same document in multiple releases (so you have multiple actions)

Reply:
Ah! Good point